src/System.IO.FileSystem/tests/FileSystemTest.cs

```diff
@@ -49,5 +49,63 @@ public abstract partial class FileSystemTest : RemoteExecutorTestBase
 
             return success;
         });
+
+        /// <summary>
+        /// Runs the given command as sudo
+        /// </summary>
+        /// <param name="commandLine">The command line to run as sudo</param>
+        /// <returns> Returns the process exit code (0 typically means it is successful)</returns>
+        protected static int RunAsSudo(string commandLine)
```

Comment:
Because public makes no sense as this isn't in a shared utility class.  Intent is to move this to the new test assembly, but wanted to allow porting to 2.0.